scripts/lib/libvirt/cleanup.py

```diff
@@ -66,7 +66,7 @@ def main():
     try:
         print("undefining {0}".format(net_name))
         network.undefine()
-    except libvirt.libvirtError:
+    except (libvirt.libvirtError, UnboundLocalError):
```

Comment:
:-1: I'd rather fix the error instead of skipping it.
http://eli.thegreenplace.net/2011/05/15/understanding-unboundlocalerror-in-python